Sources_v3/StreamChatUI/UIConfig.swift

```diff
@@ -384,3 +384,16 @@ private extension UIColor {
         }
     }
 }
+
+// Typography defined in Figma design.
+// Because our design guidelines don't really fit the `prefferedFont`, we need to redaclare this.
+public extension UIFont {
```

Comment:
Do we want to expose our fonts to user? 
I don't like when 3rd parties pollute namespaces in my project.

Perhaps we could use same approach as with colors: private extension, usage via `UIConfig.ColorPalette`?
Or overkill?

Reply:
I'm onboard with this. It'll cover the usacase "I want all titles to look like this instead"

Reply:
Thanks, for faster solution, I made it the same way as UIColor is done now. Otherwise as my 2️⃣ I will take a look how we can make `UIConfig` greater :)